### PR TITLE
[EN] rearrange sections in Skinning (intro) and rename skin.ini folder

### PR DESCRIPTION
**Pull request by MegaApplePi**

### CODE REVIEW

File: wiki/Skinning/en.md [hunk 1]
```diff
@@ -1,22 +1,106 @@
 # Skinning
 
-Skinning is one of the fundamentals of all osu! game modes.
+Skinning is one of the fundamentals of all _osu!_ game modes.
 It enables users to derive from the original skinning elements to create their own!
 
-Over the years for osu!, skinning have changed dramatically in style.
-There are various ways that you can skin osu!.
+Over the years of _osu!_, skinning have changed dramatically in style.
```
Comment (TPGPL) [inline wiki/Skinning/en.md]: skinning **has** changed

File: wiki/Skinning/en.md [hunk 2]
```diff
@@ -1,22 +1,106 @@
 # Skinning
 
-Skinning is one of the fundamentals of all osu! game modes.
+Skinning is one of the fundamentals of all _osu!_ game modes.
 It enables users to derive from the original skinning elements to create their own!
```
Comment (TPGPL) [inline wiki/Skinning/en.md]: `own ones`?

File: wiki/Skinning/en.md [hunk 3]
```diff
@@ -1,22 +1,106 @@
...
 
-Over the years for osu!, skinning have changed dramatically in style.
-There are various ways that you can skin osu!.
+Over the years of _osu!_, skinning have changed dramatically in style.
+There are various ways that you can skin _osu!_
 
-Skins vary from for-fun to a central theme (i.e anime, or group of people) to those that are nigh impossible to play with ([such as this](//osu.ppy.sh/forum/viewtopic.php?f=15&t=45689)).
-Another kind of skin are pro skins, where they maximize play field visibility and minimize visual obtrusions (i.e. replacing combobursts and hitbursts with blank images).
+Skins vary from for-fun to a central theme (e.g. anime or group of people) to those that are nigh impossible to play with (e.g. [Konata Skin](https://osu.ppy.sh/forum/t/45689)).
```
Comment (TPGPL) [inline wiki/Skinning/en.md]: bad example; change or delete

File: wiki/Skinning/en.md [hunk 4]
```diff
@@ -1,22 +1,106 @@
...
+Another type of skin are "pro" skins, where they maximize play field visibility and minimize visual obtrusions (e.g. replacing combobursts and hitbursts with blank images).
```
Comment (TPGPL) [inline wiki/Skinning/en.md]: italic `pro` instead of quoting?

Comment (TPGPL) [inline wiki/Skinning/en.md]: delete comboburst part in example; you can turn them off in osu!client

File: wiki/Skinning/en.md [hunk 5]
```diff
@@ -1,22 +1,106 @@
...
 
-*Do you want to read on the tutorial? If so, see [Skinning Tutorial](/wiki/Skinning/Tutorial/).*
+_See [Skinning Tutorial](/wiki/Skinning/Tutorial/)._
+
+## Skinning Sets
+
+If your **beatmap** uses a skin and has any of the elements included in here, all elements of the set **should be included** even if you need to use default skin elements to complete it.
+If **all** sets are included, then you may alternatively just force default skin.
+
+The **individual sets** can be found on these pages **divided into each gamemode**, the user interface and the sounds shared by most gamemodes:
```
Comment (TPGPL) [inline wiki/Skinning/en.md]: Game Modes?

File: wiki/Skinning/en.md [hunk 6]
```diff
@@ -1,22 +1,106 @@
...
+There are essentially two resolution modes _osu!_ is using.
+Each of them prefers one set of sprites.
+The first mode is **LowResolution** while the second mode is **HighResolution.**
+
+- *LowResolution mode* uses the normal sprites and ignores the HD sprites **(SD-resolution skin)**
+- *HighResolution mode* prefers HD sprites and uses normal sprites as a fallback if no HD sprite is available **(HD-resolution skin)**
+
+### osu!supporter
+
+Currently 2 sprites and 2 sounds can only be modified by [osu!supporter](/wiki/osu!supporter/) otherwise the default one will be used.
```
Comment (TPGPL) [inline wiki/Skinning/en.md]: +Currently only 2 sprites and 2 sounds can be replaced

Comment (TPGPL) [inline wiki/Skinning/en.md]: comma before `otherwise`?

File: wiki/Skinning/en.md [hunk 7]
```diff
@@ -1,22 +1,106 @@
...
+These are:
```
Comment (TPGPL) [inline wiki/Skinning/en.md]: Those are?

File: wiki/Skinning/en.md [hunk 8]
```diff
@@ -97,88 +181,6 @@ The version number, seen in the headings, is what will be used in the `Version`
...
-
-The **individual sets** can be found on these pages **divided into each gamemode** and the user interface shared by most gamemodes:
-
-- [Interface Sets](./Interface/)
-- [osu!standard Sets](./osu!/)
-- [osu!taiko Sets](./osu!taiko/)
-- [osu!catch Sets](./osu!catch/)
-- [osu!mania Sets](./osu!mania/)
-- [Sound Sets](./Sounds/)
+- does not require the `skin.ini` file
```
Comment (TPGPL) [inline wiki/Skinning/en.md]: where are other versions? 🤔

File: wiki/Skinning/en.md (hunk 2 above)
Comment (MegaApplePi) [inline wiki/Skinning/en.md]: `to create their own ones!` sounds weird

File: wiki/Skinning/en.md (hunk 3 above)
Comment (MegaApplePi) [inline wiki/Skinning/en.md]: bad example of what?

Comment (MegaApplePi) [inline wiki/Skinning/en.md]: The part of that skin being the `nigh impossible to play with` is that the hit bursts is literally just a large transparent background with a lot of stars, which in turn, making this nigh impossible to play with.

Comment (TPGPL) [inline wiki/Skinning/en.md]: you shouldnt give examples of "bad" skins; it's pretty rude to the author of it ;;

Comment (MegaApplePi) [inline wiki/Skinning/en.md]: ~~gah, we didn't say it was a bad skin, we said it was nigh impossible to play... and besides, I like that skin :p~~

fine, will rewrite to say hypothetical things

File: wiki/Skinning/en.md (hunk 8 above)
Comment (MegaApplePi) [inline wiki/Skinning/en.md]: can you explain?

Comment (MegaApplePi) [inline wiki/Skinning/en.md]: according to Haskorion's spreadsheet, _osu!_ is at `Version: 2.5`
https://docs.google.com/spreadsheets/d/1bhnV-CQRMy3Z0npQd9XSoTdkYxz0ew5e648S00qkJZ8/edit#gid=1760011283

Comment (TPGPL) [inline wiki/Skinning/en.md]: and the only explained thing is `v1.0`, not every of them

Comment (MegaApplePi) [inline wiki/Skinning/en.md]: ???? so you want me to explain the changes?

Comment (TPGPL) [inline wiki/Skinning/en.md]: otherwise it's incomplete,

Comment (MegaApplePi) [inline wiki/Skinning/en.md]: 🤔 ok..? I'm moving this section over to skin.ini tho, I will try to include the changes over there instead

**Pull request merged**